### PR TITLE
Finish plConditionalObject Python Bindings

### PR DESCRIPTION
**Pull request by Hoikas**
This pull request implements all the missing plConditionalObject subclass python bindings.

### CODE REVIEW

File: core/PRP/ConditionalObject/plAnimationEventConditionalObject.h [hunk 1]
```diff
@@ -37,6 +37,13 @@ class PLASMA_DLL plAnimationEventConditionalObject : public virtual plConditiona
 protected:
     virtual void IPrcWrite(pfPrcHelper* prc);
     virtual void IPrcParse(const pfPrcTag* tag, plResManager* mgr);
+
+public:
+    CallbackEvent getAction() const { return fAction; }
```
Comment (zrax) [inline core/PRP/ConditionalObject/plAnimationEventConditionalObject.h]: Is this enum exposed to Python anywhere?

File: core/PRP/ConditionalObject/plControlEventConditionalObject.h [hunk 2]
```diff
@@ -36,6 +36,10 @@ class PLASMA_DLL plControlEventConditionalObject : public virtual plConditionalO
 protected:
     virtual void IPrcWrite(pfPrcHelper* prc);
     virtual void IPrcParse(const pfPrcTag* tag, plResManager* mgr);
+
+public:
+    ControlEventCode getControlEvent() const { return fControlEvent; }
```
Comment (zrax) [inline core/PRP/ConditionalObject/plControlEventConditionalObject.h]: Is this enum exposed to Python anywhere?

File: core/PRP/ConditionalObject/plKeyPressConditionalObject.h [hunk 3]
```diff
@@ -36,6 +36,10 @@ class PLASMA_DLL plKeyPressConditionalObject : public virtual plConditionalObjec
 protected:
     virtual void IPrcWrite(pfPrcHelper* prc);
     virtual void IPrcParse(const pfPrcTag* tag, plResManager* mgr);
+
+public:
+    plKeyDef getKeyEvent() const { return fKeyEvent; }
```
Comment (zrax) [inline core/PRP/ConditionalObject/plKeyPressConditionalObject.h]: Is this enum exposed to Python anywhere?

File: core/PRP/ConditionalObject/plAnimationEventConditionalObject.h (hunk 1 above)
Comment (Hoikas) [inline core/PRP/ConditionalObject/plAnimationEventConditionalObject.h]: Yes, in Module.cpp

File: core/PRP/ConditionalObject/plKeyPressConditionalObject.h (hunk 3 above)
Comment (Hoikas) [inline core/PRP/ConditionalObject/plKeyPressConditionalObject.h]: I expect the user to use Python's `ord()` function

File: core/PRP/ConditionalObject/plControlEventConditionalObject.h (hunk 2 above)
Comment (Hoikas) [inline core/PRP/ConditionalObject/plControlEventConditionalObject.h]: Yes, in Module.cpp

Comment (zrax) [inline core/PRP/ConditionalObject/plControlEventConditionalObject.h]: I see CallbackEvent in there, but not the ControlEventCode values...

File: core/PRP/ConditionalObject/plKeyPressConditionalObject.h (hunk 3 above)
Comment (zrax) [inline core/PRP/ConditionalObject/plKeyPressConditionalObject.h]: That works for the ones with a direct ASCII value (e.g. 'A' or '0'), but there are also some special values (e.g. KEY_F1) which have no obvious value.

File: core/PRP/ConditionalObject/plControlEventConditionalObject.h (hunk 2 above)
Comment (Hoikas) [inline core/PRP/ConditionalObject/plControlEventConditionalObject.h]: whoops

**Comment by Hoikas** [comment]
Update to add missing enums to PyHSPlasma

**Pull request merged**